Makefile

```diff
@@ -51,6 +52,21 @@ MD_FILES := $(shell \
 		-name '*.md' \
 	| sed 's/ /\\ /g' \
 )
+UI_FILES = $(shell \
```

Comment:
Ideally we should find an automated way to filter those – to be honest I doubt keeping this list by hand will scale.